packages/dds/task-manager/src/index.ts

```diff
@@ -3,5 +3,11 @@
  * Licensed under the MIT License.
  */
 
+/**
+ * Contains a distributed data structure for queueing and processing tasks that must be completed by a single client.
```

Comment:
```suggestion
 * Contains a distributed data structure `TaskManager` to track the queues of clients that want to exclusively run tasks.
```
Maybe something like this is more accurate on what TaskManager does. I mainly want to remove the word "processing" since TaskManager plays no role in the actual execution of the task (just giving a client the "right" to begin executing)